graphdatascience/graph/graph_beta_proc_runner.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, List, Tuple, Union
```

Comment:
Moved here from `GraphProcRunner`

Reply:
The methods here are also still in `graph_proc_runner`, should they be removed there too, like those in `graph_alpha_proc_runner`?

Reply:
Good catch! `generate` should not be in `GraphProcRunner`. The others need to be in both though